crates/ruff_python_parser/src/token.rs

```diff
@@ -900,6 +900,120 @@ impl TryFrom<TokenKind> for UnaryOp {
     }
 }
 
+impl fmt::Display for TokenKind {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let value = match self {
+            TokenKind::Unknown => "<Unknown>",
+            TokenKind::StartModule => "StartProgram",
```

Comment:
Is it intentional that the display text is different from the token kind?

Reply:
No, just an oversight. I'll fix this.